bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitCommandExtension.java

```diff
@@ -206,4 +254,12 @@ private void printAccessory(String id, Console console) {
             }
         });
     }
+
+    private Collection<HomekitAccessory> getIntanceAccessories(@Nullable Integer instance) {
+        if (instance != null) {
```

Comment:
it's a shame we need two different code paths for this. Could we just refer to instance 0 in the case of non-multi-instance setups?

Reply:
OHHHH I get it now. Instance being null means "all accessories on all instances".

Any opinion on using Optional over nullable integer, instead?

Reply:
Fair enough :)